interface/src/app/gene/searchMany/gene-search-panel.tpl.html

```diff
@@ -14,4 +14,9 @@
       </search-result-table>
     </div>
   </div>
+  <div class="panel-footer">
+    <button type="button" class="btn btn-link" ng-click="switchToFew()">
+      Switch to searching with autocomplete.
+    </button>
```

Comment:
Possibly - I am going to switch stuff around and see if there is a layout that is more intuitive.